src/Kovan-DssSpell.sol

```diff
@@ -29,17 +37,131 @@ contract SpellAction {
+        // !!!!!!!! Only if PIP_RENBTC = Osm, its src is a Median and hasn't been already whitelisted due a previous deployed ilk
+        // MedianAbstract(OsmAbstract(PIP_RENBTC).src()).kiss(PIP_RENBTC);
+        // Whitelist Spotter to read the Osm data (only necessary if it is the first time the token is being added to an ilk)
+        // !!!!!!!! Only if PIP_RENBTC = Osm or PIP_RENBTC = Median and hasn't been already whitelisted due a previous deployed ilk
+        // OsmAbstract(PIP_RENBTC).kiss(MCD_SPOT);
+        // Whitelist End to read the Osm data (only necessary if it is the first time the token is being added to an ilk)
+        // !!!!!!!! Only if PIP_RENBTC = Osm or PIP_RENBTC = Median and hasn't been already whitelisted due a previous deployed ilk
+        // OsmAbstract(PIP_RENBTC).kiss(MCD_END);
+        // Set RENBTC Osm in the OsmMom for new ilk
+        // !!!!!!!! Only if PIP_RENBTC = Osm
+        OsmMomAbstract(OSM_MOM).setOsm(RENBTC_ILK, PIP_RENBTC);
+
+        // Set the global debt ceiling
+        VatAbstract(MCD_VAT).file("Line", 1234 * MILLION * RAD);
+        // Set the RENBTC-A debt ceiling
+        VatAbstract(MCD_VAT).file(RENBTC_ILK, "line", 2 * MILLION * RAD);
+        // Set the RENBTC-A dust
+        VatAbstract(MCD_VAT).file(RENBTC_ILK, "dust", 100 * RAD);
+        // Set the Lot size
+        CatAbstract(MCD_CAT).file(RENBTC_ILK, "dunk", 500 * RAD);
+        // Set the RENBTC-A liquidation penalty (e.g. 13% => X = 113)
+        CatAbstract(MCD_CAT).file(RENBTC_ILK, "chop", 113 * WAD / 100);
+        // Set the RENBTC-A stability fee (e.g. 1% = 1000000000315522921573372069)
+        JugAbstract(MCD_JUG).drip(RENBTC_ILK);
+        JugAbstract(MCD_JUG).file(RENBTC_ILK, "duty", SIX_PERCENT_RATE);
+        // Set the RENBTC-A percentage between bids (e.g. 3% => X = 103)
+        FlipAbstract(MCD_FLIP_RENBTC_A).file("beg", 103 * WAD / 100);
+        // Set the RENBTC-A time max time between bids
+        FlipAbstract(MCD_FLIP_RENBTC_A).file("ttl", 1 hours);
+        // Set the RENBTC-A max auction duration to
+        FlipAbstract(MCD_FLIP_RENBTC_A).file("tau", 1 hours);
+        // Set theRENBTC-A min collateralization ratio (e.g. 150% => X = 150)
+        SpotAbstract(MCD_SPOT).file(RENBTC_ILK, "mat", 175 * RAY / 100);
+
+        // Update RENBTC-A spot value in Vat
+        SpotAbstract(MCD_SPOT).poke(RENBTC_ILK);
+
+        // Add new ilk to the IlkRegistry
+        IlkRegistryAbstract(ILK_REGISTRY).add(MCD_JOIN_RENBTC_A);
+
+        // Set gulp amount in faucet on kovan
+        FaucetAbstract(FAUCET).setAmt(RENBTC, 10 ** 7);
+
+        // Update the changelog
         CHANGELOG.setAddress("MCD_DC_IAM", MCD_DC_IAM);
 
+        CHANGELOG.setAddress("RENBTC", RENBTC);
+        CHANGELOG.setAddress("MCD_JOIN_RENBTC_A", MCD_JOIN_RENBTC_A);
+        CHANGELOG.setAddress("MCD_FLIP_RENBTC_A", MCD_FLIP_RENBTC_A);
+        CHANGELOG.setAddress("PIP_RENBTC", PIP_RENBTC);
```

Comment:
We didn't add a new `PIP` definition for `ETH-B`, as we just reused `PIP_ETH`; however, since this could one day be replaced with its own price feed, I think this makes sense to not just use `PIP_WBTC`.

To clarify, this comment is related just to the changelog, the address for `PIP_RENBTC` should be the same as `PIP_WBTC`.